includes/common.php

```diff
@@ -81,7 +82,7 @@ function graylog_severity_label($severity)
  * @param array $command
  * @return null|string
  */
-function external_exec($command)
+function external_exec($command, $device)
```

Comment:
You can avoid this by using `DeviceCache::primary()` in the log event call below (if there is no primary device, it will send null, which is ok)

Reply:
Also, adding an argument without a default value means you need to update all the calls to that function to include the new argument.